test/integration/rack_errors_test.rb

```diff
@@ -9,10 +9,10 @@
   get '/framework_managed',   to: 'errors#framework_managed'
 end
 
-HandledException          = Class.new(StandardError)
```

Comment:
@karimmtarek All the changes in this file are unrelated to the scope of your change, please revert all of them. :smile: